families/supplychain/python/sawtooth_supplychain/client.py

```diff
@@ -195,9 +194,9 @@ def application_accept(self, record_identifier,
             SupplyChainPayload.APPLICATION_ACCEPT,
             ApplicationAcceptPayload(
                 record_identifier=record_identifier,
-                applicant=applicant,
+                applicant=applicant_public_key,
                 type=application_type,
-                timestamp=timestamp or time.time()),
+                timestamp=timestamp or int(time.time())),
```

Comment:
Based on similar type conversions present in the code, it looks like currently the granularity is seconds. Also, since the timestamp is an optional argument to the function, any design choice for the granularity (milliseconds or seconds) would have to be consistent across usages or use cases.